scipy/integrate/_ivp/ivp.py

```diff
@@ -409,6 +417,14 @@ def solve_ivp(fun, t_span, y0, method='RK45', t_eval=None, dense_output=False,
     [[2.         1.21305369 0.73534021 0.27066736 0.01350938]
      [4.         2.42610739 1.47068043 0.54133472 0.02701876]
      [8.         4.85221478 2.94136085 1.08266944 0.05403753]]
+     
+    >>> import matplotlib.pyplot as plt
```

Comment:
Doesn't really matter, but this import should not be necessary.

Reply:
Took care of it.